### ISSUE TITLE
Color contrast ratio of "Ratings" control is less than 3:1.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode#132745]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#VisualStudioCode-Win32-Apr2021;#Visual Studio Code Client;#A11ySev2;#WCAG1.4.11;#Benchmark;#DesktopApp;#Win10;#ColorContrast;

### Environment Details:
Visual Studio Code
Azure Function
Operating System: Windows 10 Pro 21H1

### Steps to Reproduce:

1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open.
4. TAB to "Ratings" controls.
5. Open the AI4D color contrast analyzer and check the color contrast ratio of the star which shows unchecked state that is filled with White color.

### Actual:
The color contrast ratio of "Rating" control that is star which shows unchecked state is 2.078:1 which is less than 3:1.
image
![image](https://user-images.githubusercontent.com/54656883/132680879-f02a16e0-cd41-4d0e-99d9-380d3ac75f51.png)

### Expected:
The color contrast ratio of "Rating" control that is star should be greater than or equal to 3:1.

### User Impact:
The low vision users won't be able to see the Rating controls properly.

**Comment by misolori** [comment]
This was already fixed in the September release (1.61) see https://github.com/microsoft/vscode/issues/133481

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;
